urllib3/_collections.py

```diff
@@ -314,8 +314,11 @@ def from_httplib(cls, message):  # Python 2
 
         for line in message.headers:
             if line.startswith((' ', '\t')):
-                key, value = headers[-1]
-                headers[-1] = (key, value + '\r\n' + line.rstrip())
+                if headers:
+                # If the first header is an invalid one, then just drop it.
+                # This mimic Python 3.
```

Comment:
That's what I can tell from testing when the first header has leading whitespace - it just disappears.